netlify.toml

```diff
@@ -0,0 +1,4 @@
+[[redirects]]
+ from = "/*"
+ to = "/index.html"
+ status = 200
```

Comment:
Great that you added the netlify.toml file so we could see your cute page not found page :)